src/NLog/Targets/MailTarget.cs

```diff
@@ -31,10 +31,9 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
-
 using JetBrains.Annotations;
 
-#if !SILVERLIGHT
+#if !SILVERLIGHT && !__ANDROID__ && !__IOS__
```

Comment:
please move those `#if !__ANDROID__ && !__IOS__` to a minimal scope. For backwards-compatibility